env0/resource_splunk_log_forwarding.go

```diff
@@ -0,0 +1,85 @@
+package env0
+
+import (
+	"context"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceSplunkLogForwarding() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceSplunkLogForwardingCreate,
+		ReadContext:   resourceSplunkLogForwardingRead,
+		UpdateContext: resourceSplunkLogForwardingUpdate,
+		DeleteContext: resourceSplunkLogForwardingDelete,
+
+		Schema: map[string]*schema.Schema{
+			"url": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The Splunk URL",
+			},
+			"token": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Sensitive:   true,
+				Description: "The Splunk token",
+			},
+			"index": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The Splunk index",
+			},
+			"audit_log_forwarding": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Whether to forward audit logs",
+			},
+			"deployment_log_forwarding": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: "Whether to forward deployment logs",
+			},
+		},
+	}
+}
+
+func resourceSplunkLogForwardingCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Should we consolidate the resources so it would be generic for the rest of the types? or is this way more inline with the way the resources are written in my codebase?